user-image/Dockerfile

```diff
@@ -28,9 +28,18 @@ RUN apt-get update && \
 	    # for soc-130ac mapping module
 	    curl \
             # for phys 151
-            gfortran && \
-    apt-get purge && apt-get clean
+            gfortran
+
+# for cogneuro connector, adding NeuroDebian repo
+RUN wget -O- http://neuro.debian.net/lists/zesty.us-ca.libre | tee /etc/apt/sources.list.d/neurodebian.sources.list
+RUN apt-key adv --recv-keys --keyserver hkp://pool.sks-keyservers.net:80 0xA5D32F012649A5A9
 
+RUN apt-get install --yes
+	    # for cogneuro connector
+	    fsl-complete \
+	    python-nipype && \
```

Comment:
This is probably not going to have any effect - installing python packages from apt is not going to be visible inside our virtual environment. They will need to be installed via pip.

Reply:
Ah ok. `fsl-complete` is not a python package, so I'll leave that there. I can use pip to install `nipype`. Initially using apt because @samyag1 had some concerns about version issues. @samyag1, does `pip install nipype==0.13.1` work here?